NPOHUBOrg/npohub-backend
# Refactor codebase to suite needs

#### What does this PR achieve?
- moves the services away from the api module
- caramelises JSON request and response
- rename assertion_helpers top test_helper_fns
- modifies decorators and code in general

#### Any background context?
- Reshaping the codebase was  necessary

#### Manual Testing Steps
- Run pytest 
- Hit register and login endpoints and see how they now require caramelised values

#### Relevant Story Links?
